### PR TITLE
#25: Pub score fixes

### PR DESCRIPTION
**Pull request by KnYaZuK**

### CODE REVIEW

File: .flutter-plugins-dependencies [hunk 1]
```diff
@@ -0,0 +1 @@
...th_provider_linux","path_provider_macos","path_provider_windows"]},{"name":"path_provider_android","dependencies":[]},{"name":"path_provider_ios","dependencies":[]},{"name":"path_provider_linux","dependencies":[]},{"name":"path_provider_macos","dependencies":[]},{"name":"path_provider_windows","dependencies":[]},{"name":"share_plus","dependencies":["url_launcher_web","url_launcher_windows","url_launcher_linux"]},{"name":"shared_preferences","dependencies":["shared_preferences_android","shared_preferences_ios","shared_preferences_linux","shared_preferences_macos","shared_preferences_web","shared_preferences_windows"]},{"name":"shared_preferences_android","dependencies":[]},{"name":"shared_preferences_ios","dependencies":[]},{"name":"shared_preferences_linux","dependencies":["path_provider_linux"]},{"name":"shared_preferences_macos","dependencies":[]},{"name":"shared_preferences_web","dependencies":[]},{"name":"shared_preferences_windows","dependencies":["path_provider_windows"]},{"name":"url_launcher_linux","dependencies":[]},{"name":"url_launcher_web","dependencies":[]},{"name":"url_launcher_windows","dependencies":[]}],"date_created":"2023-01-24 15:31:46.815016","version":"3.3.5"}
```
Comment (w3ggy) [inline .flutter-plugins-dependencies]: we can remove this file according to the first line

File: lib/src/services/demo_mode.dart [hunk 2]
```diff
@@ -14,6 +15,9 @@ class DemoMode {
     }
   }
 
+  /// Returns a `T` [DemoMode] value is [DemoMode] enabled
```
Comment (w3ggy) [inline lib/src/services/demo_mode.dart]:
```suggestion
  /// Returns a `T` [DemoMode] value if [DemoMode] is enabled
```

File: lib/src/services/proxy_manager.dart [hunk 3]
```diff
@@ -8,21 +8,25 @@ class ProxyManager {
...
   static final shared = ProxyManager._();
 
+  /// Returns `true` if proxy enabled
   Future<bool> isProxyEnabled() async {
     final prefs = await SharedPreferences.getInstance();
     return prefs.getBool(_isProxyEnabled) ?? false;
   }
 
+  /// Set the `ProxyEnabled` value to [mode] value
   Future<void> setProxyMode(bool mode) async {
```
Comment (w3ggy) [inline lib/src/services/proxy_manager.dart]:
```suggestion
  Future<void> setProxyMode(bool isProxyEnabled) async {
```

let's rename maybe?

File: .flutter-plugins-dependencies (hunk 1 above)
Comment (KnYaZuK) [reply .flutter-plugins-dependencies]: Maybe would be better to add this file to the `.gitignore`?

**Review by w3ggy — APPROVED** [review]

**Pull request merged**